crates/node/core/src/args/txpool.rs

```diff
@@ -45,6 +46,14 @@ pub struct TxPoolArgs {
     #[arg(long = "txpool.pricebump", default_value_t = DEFAULT_PRICE_BUMP)]
     pub price_bump: u128,
 
+    /// Minimum base fee required by the protocol.
+    #[arg(long = "txpool.minimal-protocol-fee", default_value_t = MIN_PROTOCOL_BASE_FEE)]
+    pub minimal_protocol_basefee: u64,
+
+    /// The max gas limit of the block
+    #[arg(long = "txpool.block-gas-limit", default_value_t = ETHEREUM_BLOCK_GAS_LIMIT)]
+    pub block_gas_limit: u64,
```

Comment:
this is slightly problematic because we also use this for op,base and I'd like to solve this differently, where do enforce this?

Reply:
hmm, this is actually a different problem and we need to make this dynamic based on the newest head. 
can we rephrase the docs, so it becomes obvious that this is the default enforced gas limit.

I'm opening an issue for the update

Reply:
https://github.com/paradigmxyz/reth/issues/10987